node-api/src/error.rs

```diff
@@ -121,7 +121,7 @@ impl RuntimeError {
                 error,
                 message: _,
             }) => {
-                let error = metadata.error(index, error)?;
+                let error = metadata.error(index, error[0])?;
                 Ok(Self::Module(PalletError {
```

Comment:
ModuleError has changed https://github.com/paritytech/substrate/blob/580ebae17fa30082604f1c9720f6f4a1cfe95b50/primitives/runtime/src/lib.rs#L476
But it seems that 0 value of error corresponds to the previous index

Reply:
For documentation purpose, relevant PR: https://github.com/paritytech/substrate/pull/10242